api/server/router/swarm/cluster_routes.go

```diff
@@ -132,7 +132,7 @@ func (sr *swarmRouter) createService(ctx context.Context, w http.ResponseWriter,
 
 	id, err := sr.backend.CreateService(service, encodedAuth)
 	if err != nil {
-		logrus.Errorf("Error creating service %s: %v", id, err)
+		logrus.Errorf("Error creating service: %v", err)
```

Comment:
It isn't possible for the name to be empty. SwarmKit validates that a service has a name before allowing it to be created.